firestore/app/src/main/java/com/google/firebase/example/fireeats/kotlin/MainFragment.kt

```diff
@@ -92,6 +94,10 @@ class MainFragment : Fragment(),
             }
         }
 
+        // MenuProvider
+        val menuHost : MenuHost = requireActivity() as MenuHost
```

Comment:
nit: the [Kotlin style guide](https://developer.android.com/kotlin/style-guide#horizontal) recommends only using a whitespace before a colon if in a class declaration.
```suggestion
        val menuHost: MenuHost = requireActivity() as MenuHost
```